packages/web-components/src/menu/menu.template.ts

```diff
@@ -0,0 +1,15 @@
+import { elements, ElementViewTemplate, html, ref, slotted } from '@microsoft/fast-element';
+import type { Menu } from './menu.js';
+
+export function menuTemplate<T extends Menu>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}">
```

Comment:
Given there will be a couple handlers, I'd suggest using: `handleMenuKeydown` as the function name here. Then for the trigger you can use `handleTriggerKeydown`.